final_prep/mod_zero_hero.rb

```diff
@@ -56,16 +96,85 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = { "name" => "Chibi the Crusher",
+                  "smell" => "wet dog",
```

Comment:
I would drop down a line for your first key. Then, you can just indent once per key.